templates/spiders/crawl.tmpl

```diff
@@ -13,14 +16,12 @@ class $classname(CrawlSpider):
         # "brand_wikidata": "$brand_wikidata",
         # "operator": "$operator",
         # "operator_wikidata": "$operator_wikidata",
-        # "extras": Categories.SHOP_XYZ.value
     }
 
     rules = (Rule(LinkExtractor(allow=r"Items/"), callback="parse_item", follow=True),)
 
     def parse_item(self, response):
-        item = {}
-        #item["domain_id"] = response.xpath('//input[@id="sid"]/@value').get()
-        #item["name"] = response.xpath('//div[@id="name"]').get()
-        #item["description"] = response.xpath('//div[@id="description"]').get()
+        item = Feature()
+        # item["name"] = response.xpath('//div[@id="name"]').get()
```

Comment:
Suggest changing this default example to `ref` rather than `name` because `ref` is more-or-less mandatory, but `name` isn't.